packages/medusa/src/services/totals.ts

```diff
@@ -531,6 +549,23 @@ class TotalsService extends TransactionBaseService {
       )
     }
 
+    const taxRate = lineItem.tax_lines.reduce((acc, next) => {
+      return acc + next.rate / 100
+    }, 0)
+    const taxAmountIncludedInPrice = !includesTax
+      ? 0
+      : Math.round(
+          calculatePriceTaxAmount({
+            price: lineItem.unit_price,
+            taxRate,
+            includesTax,
+          })
+        )
+
+    lineSubtotal =
+      (lineItem.unit_price - taxAmountIncludedInPrice) * lineItem.quantity -
+      discountAmount
+
```

Comment:
thought: round this value for good measure?

Reply:
It is rounded just after applying the rate is it not enough you think?